variables.tf

```diff
@@ -103,7 +68,18 @@ variable "agent_aws_secret_access_key" {
 }
 
 variable "castai_components_labels" {
-  type = map
+  type        = map
   description = "Optional additional Kubernetes labels for CAST AI pods"
-  default = {}
+  default     = {}
+}
+
+variable "node_configurations" {
+  type        = any
+  description = "Map of EKS node configurations to create"
+  default     = {}
+}
+
+variable "default_node_configuration" {
+  type        = string
+  description = "ID of the default node configuration"
```

Comment:
nitpick: IMO this variable should have a default - default node configuration for a cluster

Reply:
but if cluster is onboarded with node config, then we know node config id only when it is created

thats why default is set like this:

```hcl
default_node_configuration = module.cast-eks-cluster.castai_node_configurations["default"]
```